src/main/kotlin/model/CourtUsers.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Model
+import com.structurizr.model.Person
+import com.structurizr.view.ViewSet
+
+class CourtUsers private constructor() {
+  companion object : HMPPSSoftwareSystem {
```

Comment:
I've just realised we might have made `HMPPSSoftwareSystem` overly specific, as `CourtUsers` wants to conform to the same interface but isn't a software system. That's not a criticism of this PR, and shouldn't block merge, but we may want to rename the interface or create a more appropriate one – I'm guessing that would have model entities and relationships, but not views?

Reply:
👍 Raised for discussion in #52.